content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -14,7 +14,7 @@ test: no
 Follow this guide to deploy Istio and connect a virtual machine to it.
 
 {{< tip >}}
-This guide is tested and validated but note that VM support is still an alpha feature not recommended for production.
+This guide is tested and validated but note that VM support is still an experimental feature not recommended for production.
```

Comment:
This guide has been documented as `alpha` for several releases already. Seems strange to demote it to experimental in 1.9.

Seems to me that only the auto WorkloadEntry creation part is experimental, not the whole guide.

Reply:
Should be alpha, very likely to be promoted to beta in 1.9 as well.